aml-development/frontend
# fix(Submission): listing status modal

Adds reason to description text of pending_deletion state.
Adds access conditions to the modal so that users with the specific privilege levels can view the buttons

Closes #AMLOS-746

To test:
log in as bigbrother, wsmith, and jones. 
Open the listing witch doctor
Duplicate the listing. check the availability of buttons for each state the listing can be in.
bigbrother should always be able to see the buttons, Wsmith will be missing only Pending mall approval, and jones will be missing pending, pending aml, and pending deletion.
